packages/postgraphile-core/__tests__/fixtures/queries/pg10.types.graphql

```diff
@@ -0,0 +1,28 @@
+query {
+  allTypes {
+    ...typesConnection
+  }
+}
+
+fragment type on Type {
+regrole
+regnamespace
```

Comment:
Add the id here please.

```suggestion
  id
  regrole
  regnamespace
```

Reply:
Added it in, also in the `types.graphql`